update.sh

```diff
@@ -158,11 +160,19 @@ copy () {
 	update_gitignore
 }
 
+post () {
+  # execute update-kit if it exists in the update folder
+  if [ -d "update/lib/update-kit" ]; then
+    node "update/lib/update-kit"
```

Comment:
is there any intention to move the whole update script to node in the future? currently this would mean some of the update script is in one file and some is in another, different languages

Reply:
It's something we've talked about, but to be honest I think it would be quite a big job for not very much benefit...

Reply:
I think we agreed to rename the javascript part so its clearer its not the main update script

Reply:
Renamed as _update_scss